### PR TITLE
Cockpit 211

### PR DESCRIPTION
**Pull request by mvollmer**
https://mvollmer.github.io/cockpit-project.github.io/blog/cockpit-211.html

### CODE REVIEW

File: _posts/2020-01-22-cockpit-211.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+tags: cockpit linux
+slug: cockpit-211
+category: release
+summary: Cockpit with certificate improvements and more
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 211.
+
+### Better support for various certificates formats
```
Comment (martinpitt) [inline _posts/2020-01-22-cockpit-211.md]: For the tag I'm using "various TLS certificate formats" (note the singular)

File: _posts/2020-01-22-cockpit-211.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 211
+author: mvo
+date: 2020-01-22
+tags: cockpit linux
+slug: cockpit-211
+category: release
+summary: Cockpit with certificate improvements and more
```
Comment (martinpitt) [inline _posts/2020-01-22-cockpit-211.md]: maybe "with certificate improvements and layout fixes"?

File: _posts/2020-01-22-cockpit-211.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+### Better support for various certificates formats
+
+Cockpit can now read certificates in a number of formats, such as the
+one from LetsEncrypt.
+
+Specifically, the web server now accepts certificates in
+`/etc/cockpit/ws-certs.d` which have a `.crt` extension, in addition
+to `.cert`. Also, private keys can be in their own file with a `.key`
+extensions.
```
Comment (martinpitt) [inline _posts/2020-01-22-cockpit-211.md]: "extension" singular.

**Comment by martinpitt** [comment]
Release succeeded, all on [bodhi](https://bodhi.fedoraproject.org/updates/?packages=cockpit) now

File: _posts/2020-01-22-cockpit-211.md (hunk 1 above)
Comment (mvollmer) [reply _posts/2020-01-22-cockpit-211.md]: Fixed.

File: _posts/2020-01-22-cockpit-211.md (hunk 2 above)
Comment (mvollmer) [reply _posts/2020-01-22-cockpit-211.md]: Yes.

File: _posts/2020-01-22-cockpit-211.md (hunk 3 above)
Comment (mvollmer) [reply _posts/2020-01-22-cockpit-211.md]: Fixed.

File: _posts/2020-01-22-cockpit-211.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 211.
+
+### Better support for various TLS certificate formats
+
+Cockpit can now read certificates in a number of formats, such as the
+one from LetsEncrypt.
+
+Specifically, the web server now accepts certificates in
+`/etc/cockpit/ws-certs.d` which have a `.crt` extension, in addition
+to `.cert`. Also, private keys can be in their own file with a `.key`
```
Comment (allisonkarlitskaya) [inline _posts/2020-01-22-cockpit-211.md]: I'd say "can now be in a separate file" to emphasize the difference here.

Comment (mvollmer) [reply _posts/2020-01-22-cockpit-211.md]: Very good!

**Pull request merged**